core/src/main/scala/org/apache/spark/rdd/PairRDDFunctions.scala

```diff
@@ -1014,7 +1015,7 @@ class PairRDDFunctions[K, V](self: RDD[(K, V)])
    * Output the RDD to any Hadoop-supported file system, using a Hadoop `OutputFormat` class
    * supporting the key and value types K and V in this RDD.
    *
-   * Note that, we should make sure our tasks are idempotent when speculation is enabled, i.e. do
+   * @note We should make sure our tasks are idempotent when speculation is enabled, i.e. do
```

Comment:
Likewise I wonder if these following sentences all go into the note? they go together.